typescript/src/entities/user.ts

```diff
@@ -95,120 +96,128 @@ export class User extends Entity {
+  @User.property({ arrayType: "Assignment" })
   public assignments?: Array<Assignment>;
 
-  @User.property({arrayType: "UserCompany"})
+  @User.property({ arrayType: "UserCompany" })
   public userCompanies?: Array<UserCompany>;
 
-  @User.property({arrayType: "Draft"})
+  @User.property({ arrayType: "Draft" })
   public drafts?: Array<Draft>;
 
-  @User.property({arrayType: "CompanyInvitation"})
+  @User.property({ arrayType: "CompanyInvitation" })
   public companyInvitations?: Array<CompanyInvitation>;
 
-  @User.property({arrayType: "Address"})
+  @User.property({ arrayType: "Address" })
   public addresses?: Array<Address>;
 
-  @User.property({arrayType: "MerchiFile"})
+  @User.property({ arrayType: "MerchiFile" })
   public uploadFiles?: Array<MerchiFile>;
 
-  @User.property({arrayType: "JobComment"})
+  @User.property({ arrayType: "JobComment" })
   public forwardedJobComments?: Array<JobComment>;
 
-  @User.property({arrayType: "Job"})
+  @User.property({ arrayType: "Job" })
   public appliedJobs?: Array<Job>;
 
-  @User.property({arrayType: "Job"})
+  @User.property({ arrayType: "Job" })
   public managedJobs?: Array<Job>;
 
-  @User.property({arrayType: "Job"})
+  @User.property({ arrayType: "Job" })
   public draftingJobs?: Array<Job>;
 
-  @User.property({arrayType: "Product"})
+  @User.property({ arrayType: "Product" })
   public saved_products?: Array<Product>;
 
-  @User.property({arrayType: "Cart"})
+  @User.property({ arrayType: "Cart" })
   public carts?: Array<Cart>;
 
-  @User.property({arrayType: "Payment"})
+  @User.property({ arrayType: "Payment" })
   public payments?: Array<Payment>;
 
-  @User.property({arrayType: "EnrolledDomain"})
+  @User.property({ arrayType: "EnrolledDomain" })
   public enrolledDomains?: Array<EnrolledDomain>;
 
-  @User.property({arrayType: "Invoice"})
+  @User.property({ arrayType: "Invoice" })
   public responsibleInvoices?: Array<Invoice>;
 
-  @User.property({arrayType: "Invoice"})
+  @User.property({ arrayType: "Invoice" })
   public createdInvoices?: Array<Invoice>;
 
-  @User.property({arrayType: "Invoice"})
+  @User.property({ arrayType: "Invoice" })
   public invoicesHas?: Array<Invoice>;
 
-  @User.property({arrayType: "DomainInvitation"})
+  @User.property({ arrayType: "DomainInvitation" })
   public sentDomainInvitations?: Array<DomainInvitation>;
 
-  @User.property({arrayType: "DomainInvitation"})
+  @User.property({ arrayType: "DomainInvitation" })
   public receivedDomainInvitations?: Array<DomainInvitation>;
 
-  @User.property({arrayType: "Theme"})
+  @User.property({ arrayType: "Theme" })
   public themes?: Array<Theme>;
 
-  @User.property({arrayType: "ProductionComment"})
+  @User.property({ arrayType: "ProductionComment" })
   public forwardedProductionComments?: Array<ProductionComment>;
+
+  public userLocalTimeFormat = (time: Date, formatString?: string): string => {
+    var format = formatString ? formatString : "ddd Do MMM YY";
```

Comment:
formatting times for display is a display issue. it probably belongs in a view layer and not an api wrapper layer

Reply:
> formatting times for display is a display issue. it probably belongs in a view layer and not an api wrapper layer

this is very true, however whether sdk is only a api wrapper is questionable. If we do not provide this thing in sdk, someone has implemented it in view level, which to me is very human, as it is our decision to make our time data timezone aware.  Sdk user should know as less as possible, also there should be function lays in sdk to indicate how to proper formating time.


In short, telling sdk user "we give you a UTC time and give you timezone feel free to format the time by yourself, sorry do you know how to convert? sorry you do not deserve to use this sdk" is not nature to me.